tracing-subscriber/src/fmt/mod.rs

```diff
@@ -1188,10 +1188,13 @@ pub fn try_init() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
 /// If the `tracing-log` feature is enabled, this will also install
 /// the LogTracer to convert `Log` records into `tracing` `Event`s.
 ///
-/// This is shorthand for
+/// If the `env-filter` feature is enabled, this is shorthand for
```

Comment:
nit, take it or leave it: it might be worth linking to the list of feature flags in the top-level docs...but the previous comment mentioning the `tracing-log` feature doesn't do that, so we can address this separately.